### DISCUSSION TITLE
Support for 0.180.0+ strawberry-graphql

### DISCUSSION THREAD

**Discussion by nickqweaver** [nrbnlulu/strawberry-django-auth#397] [General]
Hey just curious if/when support will be added for `strawberry-graphql` latest? There were some [breaking changes](https://strawberry.rocks/docs/breaking-changes/0.180.0) in 0.180.0 regarding changing `context` to a `dict`.

**Comment by nrbnlulu** [comment]
Sure maybe tomorrow I'll have time
